From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 7 Jan 2009
Subject: Disable all cgroups
References: bnc#436025
Patch-mainline: no

It causes speed regressions. Disable them by default.

Users are now able to enable it by kernel cmdline parameter.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/cgroup.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/kernel/cgroup.c
+++ b/kernel/cgroup.c
@@ -2530,6 +2530,8 @@ int __init cgroup_init_early(void)
 			BUG();
 		}
 
+		ss->disabled = 1; /* performance regressions */
+
 		if (ss->early_init)
 			cgroup_init_subsys(ss);
 	}
